demo/vagrant/Vagrantfile

```diff
@@ -25,7 +25,7 @@ sudo docker --version
 sudo apt-get install unzip curl vim -y
 
 echo "Installing Nomad..."
-NOMAD_VERSION=0.10.4
+NOMAD_VERSION=0.10.5
```

Comment:
Nomad 0.11.0 is out. Let's bump it to that version!

Reply:
yup. Thanks! It's done now :)